### PR TITLE
feat(contact2): Style page

### PR DESCRIPTION
**Pull request by damassi**
Styles new contact 2 page: 

contact

### CODE REVIEW

**Review by dzucconi — APPROVED** [review]
I find the layout a bit awkward in our grid (probably should just simplify). But let's go.

File: src/v2/Apps/_Contact2/ContactApp.tsx [hunk 1]
```diff
@@ -1,9 +1,153 @@
...
+                  For pitches and story ideas, contact{" "}
+                  <a href="mailto:pitches@artsy.net">pitches@artsy.net</a>{" "}
+                  <br />
+                  <br />
+                  For comments and questions, contact{" "}
+                  <a href="mailto:comments@artsy.net">
+                    comments@artsy.net
+                  </a>{" "}
+                  <br />
+                  <br />
```
Comment (dzucconi) [inline src/v2/Apps/_Contact2/ContactApp.tsx:81]: :\

Comment (damassi) [reply src/v2/Apps/_Contact2/ContactApp.tsx:81]: What is a better way to do this (I honestly don't know)

Comment (damassi) [reply src/v2/Apps/_Contact2/ContactApp.tsx:81]: p tags?

File: src/v2/Apps/_Contact2/ContactApp.tsx [hunk 2]
```diff
@@ -1,9 +1,153 @@
...
 
 export const ContactApp: React.FC = () => {
   return (
     <>
-      <Text variant="xl">Contact App</Text>
+      <MetaTags
+        title="Contact Us"
+        description="We're here to help. Contact an Artsy team member about Sales, Auctions, Press inquiries, Editorial, Careers, and more."
+        pathname="/contact"
+      />
```
Comment (damassi) [inline src/v2/Apps/_Contact2/ContactApp.tsx:11]: how we do page header stuff

File: src/v2/Apps/_Contact2/ContactApp.tsx [hunk 3]
```diff
@@ -1,9 +1,153 @@
...
+const Item: React.FC<{
+  title?: string | undefined
+  content: JSX.Element | string
+}> = ({ title, content }) => {
+  return (
+    <Column span={6} my={[1, 0]}>
+      <Text variant={["md", "lg"]} mb={1} fontWeight={["bold", "normal"]}>
+        {title}
+      </Text>
+      <Text variant={["sm", "md"]}>{content}</Text>
```
Comment (dzucconi) [inline src/v2/Apps/_Contact2/ContactApp.tsx:150]: Just use `sm`

File: src/v2/Apps/_Contact2/ContactApp.tsx [hunk 4]
```diff
@@ -1,9 +1,153 @@
...
+  )
+}
+
+const Item: React.FC<{
+  title?: string | undefined
+  content: JSX.Element | string
+}> = ({ title, content }) => {
+  return (
+    <Column span={6} my={[1, 0]}>
+      <Text variant={["md", "lg"]} mb={1} fontWeight={["bold", "normal"]}>
```
Comment (dzucconi) [inline src/v2/Apps/_Contact2/ContactApp.tsx:147]: Is the bold necessary?

File: src/v2/Apps/_Contact2/ContactApp.tsx (hunk 1 above)
Comment (dzucconi) [reply src/v2/Apps/_Contact2/ContactApp.tsx:81]:
```
<Text as="p" variant="sm" mb={1}>or whatever</Text>
```

File: src/v2/Apps/_Contact2/ContactApp.tsx [hunk 5]
```diff
@@ -1,9 +1,153 @@
...
     </>
   )
 }
+
+const Section: React.FC<{ title: string; content: JSX.Element }> = ({
+  title,
+  content,
+}) => {
+  return (
+    <GridColumns width={["100%", "70%"]} margin="auto" py={4}>
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/v2/Apps/_Contact2/ContactApp.tsx:130]: `py` = padding y = padding top & padding bottom. 

Palette has this nice short-hand way of writing common things. Other examples:

`m` = margin 
`mr` = margin right, etc.

File: src/v2/Apps/_Contact2/ContactApp.tsx (hunk 4 above)
Comment (damassi) [reply src/v2/Apps/_Contact2/ContactApp.tsx:147]: nah

**Pull request merged**